x/axelarnet/keeper/msg_server.go

```diff
@@ -337,6 +335,13 @@ func (s msgServer) RouteIBCTransfers(c context.Context, _ *types.RouteIBCTransfe
 			continue
 		}
 
+		pathSplit := strings.SplitN(path, "/", 2)
+		if len(pathSplit) != 2 {
+			s.Logger(ctx).Error(fmt.Sprintf("invalid path %s for chain %s", path, chain.Name))
+			continue
+		}
```

Comment:
is there a reason we can't get it already parsed from the keeper?

Reply:
we can, don't want keeper to handle logic. We can create a type to parse and validate it?